src/object/schedcontrol.c

```diff
@@ -23,23 +23,6 @@ static exception_t invokeSchedControl_Configure(sched_context_t *target, word_t
         /* remove from scheduler */
         tcbReleaseRemove(target->scTcb);
         tcbSchedDequeue(target->scTcb);
-        /* bill the current consumed amount before adjusting the params */
```

Comment:
This is larger than the change I had in mind for issue 2. I'll have another look what exactly I meant there.

Reply:
Ok, I get it now, this is still part of issue 3, and removes code that is now unreachable. All good.